db/schema.rb

```diff
@@ -11,16 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160817012546) do
+ActiveRecord::Schema.define(version: 20160817154616) do
 
   create_table "tutorials", force: :cascade do |t|
     t.string   "title"
     t.string   "tagline"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
     t.integer  "user_id"
     t.integer  "impressions"
     t.text     "content"
+    t.string   "author"
```

Comment:
Are any of your views going to break if either of these are nil?  A model validation may be in order.